sigs/fedora.md

```diff
@@ -0,0 +1,12 @@
+# Konflux + Fedora Special Interest Group
+
+## Purpose:
+
+* Operate and manage the [Fedora Konflux instance](https://gist.github.com/ralphbean/a3644111a549e8cedb0b207f90d42dc9).
```

Comment:
If this is a public good instance that the community is supporting, can we move this content into the org instead of on a private gist?

Reply:
ack.